enclave-runtime/Cargo.lock

```diff
@@ -3433,7 +3433,7 @@ version = "1.6.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "1f559b464de2e2bdabcac6a210d12e9b5a5973c251e102c44c585c71d51bd78e"
 dependencies = [
- "cfg-if 1.0.0",
+ "cfg-if 0.1.10",
```

Comment:
What is the reason for this downgrade?

Reply:
I guess so.

Reply:
@clangenb is correct here - the only thing I did was to run cargo update -p sha2
But I think sha2 should depend on cfg-if 1.0.0 : https://github.com/RustCrypto/hashes/blob/a7431da030b14ff144a30638c7cd3604befc1512/sha2/Cargo.toml#L21

I'll try to run an update again

Reply:
Fixed :)